spring-pulsar-docs/src/main/antora/modules/ROOT/pages/reference/testing-applications.adoc

```diff
@@ -1,42 +1,46 @@
 [[testing-applications]]
 = Testing Applications
+
```

Comment:
When there is a blank line it prevents it from showing the TOC. 
Once we publish the module then we can link this awesome document you added to the navbar.adoc.